client/app/shared/modals/data-upload/data-upload.component.ts

```diff
@@ -71,8 +84,12 @@ export class DataUpload {
     this.fileHovering = false
   }
 
+  public open () {
+    jQuery('.data-upload').modal().modal('show')
```

Comment:
You might not need `jQuery`: http://youmightnotneedjquery.com/

Reply:
This is actually a semantic function... https://semantic-ui.com/modules/modal.html